spirv_common.hpp

```diff
@@ -343,14 +343,16 @@ struct SPIREntryPoint
 {
 	SPIREntryPoint(uint32_t self_, spv::ExecutionModel execution_model, std::string entry_name)
```

Comment:
Make this const& since you're not moving from it anymore.

Reply:
Done.